common/src/backends/trace.rs

```diff
@@ -80,8 +80,9 @@ where
     subscriber
         .with(filter_layer)
         .with(fmt_layer)
-        .with(otel_layer)
         .with(appender_tracing_layer)
+        .with(otel_layer)
+        .with(ErrorTracingLayer::new())
```

Comment:
will this end up being used by `shuttle-runtime`?

Reply:
The runtime doesn't use `setup_tracing`